app/controllers/idv/in_person/ready_to_verify_controller.rb

```diff
@@ -10,7 +10,12 @@ class ReadyToVerifyController < ApplicationController
       before_action :confirm_in_person_session
 
       def show
-        analytics.idv_in_person_ready_to_verify_visit
+        bucket = session[:in_person_cta_variant]
+        if bucket
+          analytics.idv_in_person_ready_to_verify_visit(in_person_cta_variant: bucket)
```

Comment:
can we move bucket to a separate method and then merge this with the `extra` param and then pass in extra_analytics_attributes? so maybe something like: 
https://github.com/18F/identity-idp/blob/main/app/controllers/idv/cancellations_controller.rb#L46-56

Reply:
I wasn't aware of this pattern before - great! Getting this implemented.